src/macaron/slsa_analyzer/analyzer.py

```diff
@@ -72,6 +72,12 @@ def __init__(self, output_path: str, build_log_path: str) -> None:
             logger.error("Cannot start the analysis. Exiting ...")
             sys.exit(1)
 
+        logger.info(
+            "The following checks are excluded from user configuration: %s",
```

Comment:
```suggestion
            "The following checks are excluded based on the user configuration: %s",
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/254/commits/7c94792b6406387beec45d59fcbc2002323e8491